snap/quota/quota.go

```diff
@@ -238,6 +243,19 @@ func (grp *Group) CurrentTaskUsage() (int, error) {
 	return int(count), nil
 }
 
+// IsSnapRelated returns true if a snap is a part of any parent group
+// or the current quota group.
+func (grp *Group) IsSnapRelated(snap string) bool {
+	i := grp
```

Comment:
why i as the var name?

Reply:
I think that was the C-developer in me, I use 'i' for 'iterator'. I'll rename it to 'currentGrp'